src/oc/web/actions.cljs

```diff
@@ -887,32 +887,72 @@
   (api/get-comments activity-uuid)
   (assoc db :comment-add-finish true))
 
-(defmethod dispatcher/action :reaction-toggle
-  [db [_ activity-uuid reaction-data]]
+(defn- current-board-key
```

Comment:
Maybe we should move this function to `oc.web.dispatcher`? it already contains all the key functions: `all-posts-key` and `board-data-key` it would make sense to have this there too.
Dispatcher already import `router` so it won't be a problem.
Anyway, your call if you want to move this now or in another branch.